torch/utils/_sympy/reference.py

```diff
@@ -0,0 +1,122 @@
+import sympy
+
+# The normal Python interpretation of the operators
+# NB: For magic methods this needs to use normal magic methods
+# so that test_magic_methods works
+class ReferenceAnalysis:
```

Comment:
I think most of these could be more succinct as a mapping from name to operator. E.g. `ge -> operator.ge`.

Reply:
do you actually want them metaprogrammed? It seems easier to understand if we don't